Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -330,6 +330,12 @@ - (NSMutableDictionary *)mapContextValues:(NSDictionary *)properties context:(NS
         NSMutableDictionary *data = [[NSMutableDictionary alloc] initWithCapacity:contextValuesSize];
         NSDictionary *contextValues = self.settings[@"contextValues"];
         for (NSString *key in contextValues) {
+            if ([key hasPrefix:@"traits."]){
+                NSDictionary *contextTraits = [context valueForKey:@"traits"];
+                NSArray *arrayOfKeyComponents = [key componentsSeparatedByString:@"."];
+                NSString *parsedKey = arrayOfKeyComponents[1];
```

Comment:
Do we intend to support nested traits? e.g.; `traits.person.name`

Reply:
Not at this time no. This is a specific ask for Fox to support traits within the context object. Based off their POC testing all traits have been top level in `traits.context`.